evap/staff/tests/test_forms.py

```diff
@@ -588,6 +588,79 @@ def test_prevent_contribution_deletion_with_answers(self):
         self.assertFalse(formset.forms[0].show_delete_button)
         self.assertTrue(formset.forms[1].show_delete_button)
 
+    def test_answers_for_removed_questionnaires_deleted(self):
+        # pylint: disable=too-many-locals
+        semester = baker.make(Semester)
+        evaluation = baker.make(
+            Evaluation,
+            course=baker.make(Course, semester=semester),
+        )
```

Comment:
```suggestion
        evaluation = baker.make(Evaluation)
```
or am I missing something?